integration_tests/test_openapi.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from robyn import OpenAPI
+from robyn.responses import FileResponse
+
+
+@pytest.fixture
+def openapi_instance():
+    return OpenAPI()
+
+
+@pytest.fixture
+def subrouter_openapi_instance():
+    return OpenAPI()
+
+
+def test_openapi_initialization(openapi_instance):
+    spec = openapi_instance.openapi_spec
+    assert spec["openapi"] == "3.0.0"
+    assert spec["info"]["title"] == "Robyn API"
+    assert spec["info"]["version"] == "1.0.0"
+    assert "paths" in spec
+    assert "components" in spec
+    assert "servers" in spec
+    assert "externalDocs" in spec
+
+
+def test_docs_handler(openapi_instance):
+    html_response = openapi_instance.docs_handler()
+    assert isinstance(html_response, FileResponse)
+
+
+def test_json_handler(openapi_instance):
+    json_response = openapi_instance.json_handler()
+    assert isinstance(json_response, str)
+    assert "openapi" in json_response
+    assert "info" in json_response
+    assert "paths" in json_response
+    assert "components" in json_response
+    assert "servers" in json_response
+    assert "externalDocs" in json_response
+
+
+def test_add_openapi_path(openapi_instance):
```

Comment:
These are all unit tests. I don't like to keep unit tests in the repo,and I don't think these would be any different. The tests should be written in an integrated test format like done for all the other routes.